maplibre_gl/ios/maplibre_gl/Sources/maplibre_gl/MapLibreMapController.swift

```diff
@@ -56,9 +56,9 @@ class MapLibreMapController: NSObject, FlutterPlatformView, MLNMapViewDelegate,
             target: self,
             action: #selector(handleMapTap(sender:))
         )
-        for recognizer in mapView.gestureRecognizers! where recognizer is UITapGestureRecognizer {
+        /*for recognizer in mapView.gestureRecognizers! where recognizer is UITapGestureRecognizer {
```

Comment:
Why did you comment it out?

Reply:
My bad, this was to enhanced the speed of the callback, with this commented the interaction callback switch from 1s to 0.1s. I was unable to remove all the slow gestureRecognizers

Reply:
Workaround is still there — did you find another reason to keep it?

Reply:
No, but I haven't been able to find a good solution for this important delay. Perhaps you have an idea?